rotors_gazebo/launch/iris_outdoor_world.launch

```diff
@@ -0,0 +1,22 @@
+<launch>
+  <arg name="enable_logging" default="false"/>
+  <arg name="enable_ground_truth" default="true"/>
+  <arg name="log_file" default="iris"/>
+  <arg name="headless" default="false"/>
+  <arg name="gui" default="true"/>
+
+  <env name="GAZEBO_MODEL_PATH" value="$(find drcsim_model_resources)/gazebo_models"/>
```

Comment:
We are not currently depending on any drcsim stuff, so either we move this to https://github.com/ethz-asl/rotors_simulator_demos or we copy the necessary files to this repo, I would suggest.